src/uu/dd/src/dd_unit_tests/block_unblock_tests.rs

```diff
@@ -0,0 +1,357 @@
+/* cspell:disable */
+
+use super::*;
+
+const NL: u8 = '\n' as u8;
+const SPACE: u8 = ' ' as u8;
```

Comment:
```suggestion
const NL: u8 = b'\n';
const SPACE: u8 = b' ';
```
Nit: you can make ASCII characters bytes with `b`.

Reply:
:+1: